### PR TITLE
API-22809/ DocServer: API versions last updated should include dependants updates

### PR DESCRIPTION
**Pull request by mheusgit**
- Updated filtering for renderConfig, loastConfig, and gatewayConfig to stop filtering out the updatedAt property

### CODE REVIEW

File: src/api/gatewayConfig/gatewayConfig.interface.ts [hunk 1]
```diff
@@ -10,4 +10,5 @@ export interface GatewayConfigResponse {
   routeId?: string;
   removePath?: string;
   exposedRoute?: string;
+  updatedAt: Date;
```
Comment (christineyost) [inline src/api/gatewayConfig/gatewayConfig.interface.ts:13]: Should this be optional like in the other response interfaces?

Comment (mheusgit) [reply src/api/gatewayConfig/gatewayConfig.interface.ts:13]: It should be.

Comment (mheusgit) [reply src/api/gatewayConfig/gatewayConfig.interface.ts]: Updated

**Review by christineyost — APPROVED** [review]
Looks good to me!

**Review by dsteeve — APPROVED** [review]
lgtm!

**Pull request merged**